subprojects/docs/src/samples/userguide/tutorial/loggingKts/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+// tag::use-slf4j[]
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+Logger slf4jLogger = LoggerFactory.getLogger("some-logger")
+slf4jLogger.info("An info log message logged using SLF4j")
+// end::use-slf4j[]
+
+// tag::use-logger[]
+logger.quiet("An info log message which is always logged.")
+logger.error("An error log message.")
+logger.warn("A warning log message.")
+logger.lifecycle("A lifecycle info log message.")
+logger.info("An info log message.")
+logger.debug("A debug log message.")
+logger.trace("A trace log message.")
+// end::use-logger[]
+
+// tag::use-logger-placeholder[]
+logger.info("A {} log message", "info")
+// end::use-logger-placeholder[]
+
+// tag::use-println[]
+println("A message which is logged at QUIET level")
+// end::use-println[]
+
+// tag::capture-stdout[]
+logging.captureStandardOutput(LogLevel.INFO)
+println("A message which is logged at INFO level")
+// end::capture-stdout[]
+
+// tag::task-capture-stdout[]
+tasks.create("logInfo") {
```

Comment:
I think this should be something like:

```
tasks {
    val logInfo by registering {
        logging.captureStandardOutput(LogLevel.INFO)
        doFirst {
           println("A task message which is logged at INFO level")
        }
    }
}
```

Reply:
I don't think we want to use delegated properties syntax here, but we might want to use the `tasks {}` block. WDYT @eskatos?

Reply:
We should try make the Groovy/Kotlin samples as similar as possible.

The behavior *must* be the same, so no lazy configuration api usage when the Groovy sample is not using lazy configuration.

FWIW, the strict equivalent of the Groovy:
```groovy
task logInfo {
    logging.captureStandardOutput LogLevel.INFO
    doFirst {
        println 'A task message which is logged at INFO level'
    }
}
```

would be the Kotlin:
```kotlin
task("logInfo") {
    logging.captureStandardOutput(LogLevel.INFO)
    doFirst {
        println("A task message which is logged at INFO level")
    }
}
```

Reply:
Delegated properties are useful when one need a reference to the created/configured task for later use in the same script.